src/main/java/com/conveyal/osmlib/VanillaExtract.java

```diff
@@ -130,6 +131,9 @@ public void service(Request request, Response response) throws Exception {
                 tileSource.copyTo(sink);
                 response.setStatus(HttpStatus.OK_200);
             } catch (Exception ex) {
+                String badFormatMessage = "URI format: /min_lat,min_lon,max_lat,max_lon[.pbf|.vex] (all coords in decimal degrees)\n";
```

Comment:
Why is a string containing a newline being substituted into a log message? I think maybe badFormatMessage is intended to be the body of the response.